beater/connections.go

```diff
@@ -45,8 +59,10 @@ func getSocketInfoFromTcpDiag(pollInterval time.Duration, socketInfo chan<- *soc
 	}
 }
 
-func getSocketInfo(enableTcpDiag bool, pollInterval time.Duration, socketInfo chan<- *sockets.SocketInfo) {
-	if enableTcpDiag {
+func getSocketInfo(enableDocker, enableTcpDiag bool, pollInterval time.Duration, socketInfo chan<- *sockets.SocketInfo) {
+	if enableDocker {
+		getSocketInfoFromDocker("unix:///var/run/docker.sock", pollInterval, socketInfo)
```

Comment:
I don't think the docker socket is ever somewhere else, but in the future we might want to allow connecting over TCP+TLS. For that to work we'd also need a way to configure the right certificates etc, though.